contracts/optimistic-ethereum/mockOVM/verification/mockOVM_BondManager.sol

```diff
@@ -70,7 +70,7 @@ contract mockOVM_BondManager is iOVM_BondManager, Lib_AddressResolver {
         )
     {
         // Only authenticate sequencer to submit state root batches.
-        return _who == resolve("OVM_Sequencer");
+        return _who == resolve("OVM_Proposer");
```

Comment:
This is a notable redeploy requirement; could throw us for a loop if not careful @tynes

Reply:
Not a full redeploy though. Just need to change the address in the manager.